server/routes/money.js

```diff
@@ -155,6 +239,72 @@ const createMoneyRouter = ({
     }
   });
 
+  router.get('/transactions/private', async (req, res, next) => {
+    try {
+      const templateData = {
+        title: 'Your transactions',
+        config: {
+          content: false,
+          header: false,
+          postscript: true,
+          detailsType: 'small',
+          returnUrl: req.originalUrl,
+        },
+      };
+
+      const { user } = req;
+
+      if (user) {
+        const { selectedDate } = req.query;
+        const { dateSelection, fromDate, toDate } = processSelectedDate(
+          selectedDate,
+        );
+
+        const transactionsData = await prisonerInformationService.getPrivateTransactionsFor(
+          user,
+          fromDate,
+          toDate,
+        );
+
+        if (!transactionsData) {
+          return next('Failed to fetch transaction data');
```

Comment:
does the error handler support a string error?

Reply:
Good point, no it does not. Fixed